pkg/crc/machine/bundle/extract.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func Extract(sourcepath string) (*CrcBundleInfo, error) {
-	file, err := os.Open(sourcepath)
+	file, err := os.Open(filepath.Clean(sourcepath))
```

Comment:
Is this going to break `crc start -b ../mybundle` ?

Reply:
No, it's not going to break this, https://play.golang.org/p/fkbwXEnI3sw